person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def rent_book(rental)
+    @rentals << rental unless @rentals.includes?(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] **[OPTIONAL]** If you intended for this method to be like the `add_rentals` one in the Book class I would suggest you refactor it to look like it or create a similar one.